cmd/sgx_plugin/README.md

```diff
@@ -142,4 +205,69 @@ I0626 20:33:01.414446  964346 server.go:219] Start server for provision at: /var
 I0626 20:33:01.414640  964346 server.go:219] Start server for enclave at: /var/lib/kubelet/device-plugins/sgx.intel.com-enclave.sock
 I0626 20:33:01.417315  964346 server.go:237] Device plugin for provision registered
 I0626 20:33:01.417748  964346 server.go:237] Device plugin for enclave registered
-```
\ No newline at end of file
+```
+
+### SGX device plugin demos
+#### SGX ECDSA Remote Attestation
+
+##### Remote Attestation Prerequisites
+
+For the SGX ECDSA Remote Attestation demos to work, the platform must be correctly registered and an
+SGX PCK Certificate Cache Service running.
+
+For more information, refer to:
+[Intel® Software Guard Extensions (Intel® SGX) Services](https://api.portal.trustedservices.intel.com/) and
+[Intel® Software Guard Extensions SDK for Linux](https://01.org/intel-software-guard-extensions)
+
+The demo guides to run a single-node kubernetes cluster with a PCCS servicing localhost connections.
+
+Furthermore, the Kubernetes cluster must be set up according the [instructions above](#pre-built-images).
+
+##### Build the image
+
+The demo uses container images build from Intel® SGX SDK and DCAP releases.
+
+To build the demo images:
+
+```bash
+$ cd ${INTEL_DEVICE_PLUGINS_SRC}
+$ make sgx-aesmd-demo
```

Comment:
I don't see these make targets. Does this depend on another PR?

Reply:
`sgx-aesmd-demo` is now in master.